vyxal/elements.py

```diff
@@ -1011,7 +1011,30 @@ def cosine(lhs, ctx):
 def count_item(lhs, rhs, ctx):
     """Element O
     (any, any) -> returns the number of occurances of b in a
+    (any, fun) -> all elements in a where the result of b(x) is highest
     """
+    if vy_type(lhs) == types.FunctionType:
+        lhs, rhs = rhs, lhs
+    if vy_type(rhs) == types.FunctionType:
+        lhs = iterable(lhs, ctx=ctx)
+        if not lhs:
+            return lhs
+        m = None
+        fun_vals = []
+        for item in lhs:
```

Comment:
won't finish for infinite lists

Reply:
@Lyxal It's impossible for a "maximum"/"maximums" element to work with infinite lists, so that's fine